app/lib/frontend/static_files.dart

```diff
@@ -13,7 +13,7 @@ import 'package:pana/pana.dart' show runProc;
 import 'package:path/path.dart' as path;
 
 const String _defaultStaticPath = '/static';
-const _staticRootPaths = <String>['favicon.ico', 'robots.txt'];
+const _staticRootPaths = <String>['favicon.ico', 'robots.txt', 'osd.xml'];
```

Comment:
`/favicon.ico` and `/robots.txt` are exceptions because they are expected to be there. For `osd.xml` there is no such expectation, let it go under `/static/osd.xml` (and you can remove it from here).